numba/core/runtime/_nrt_python.c

```diff
@@ -286,7 +286,7 @@ PyObject* try_to_return_parent(arystruct_t *arystruct, int ndim,
 }
 
 NUMBA_EXPORT_FUNC(PyObject *)
-NRT_adapt_ndarray_to_python(arystruct_t* arystruct, int ndim,
+NRT_adapt_ndarray_to_python_acqref(arystruct_t* arystruct, int ndim,
```

Comment:
Also suggest adding a docstring, perhaps ref this PR?

Reply:
done